my-app/src/styled.ts

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components"
+
+export const StyledWrapper = styled.div<{className: 'wrapper', theme: 'light' | 'dark'}>`
```

Comment:
Why do we need a className here? If you want to set a className to `StyledWrapper` - just do it like that:

`<StyledWrapper className="wrapper" />`